src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectCreateChangeRequestForm.tsx

```diff
@@ -0,0 +1,270 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import {
+  Autocomplete,
+  AutocompleteRenderInputParams,
+  Box,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Grid,
+  MenuItem,
+  RadioGroup,
+  Select
+} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import NERSuccessButton from '../../../components/NERSuccessButton';
+import * as yup from 'yup';
+import { Controller, useFieldArray, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { ChangeRequestReason, ChangeRequestType } from 'shared';
+import { TextField, IconButton } from '@mui/material';
+import { wbsTester } from '../../../utils/form';
+import ReactHookTextField from '../../../components/ReactHookTextField';
+import { NERButton } from '../../../components/NERButton';
+import { ChangeEvent } from 'react';
+import { ProjectCreateChangeRequestFormInput } from './ProjectEditContainer';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { ProjectFormInput } from './ProjectForm';
+import { CreateChangeRequestFormInput } from '../../CreateChangeRequestPage/CreateChangeRequest';
+
+interface ProjectCreateChangeRequestFormProps {
+  onSubmit: (data: ProjectCreateChangeRequestFormInput) => void;
+  open: boolean;
+  onClose: () => void;
+  projectEdits?: ProjectFormInput;
+}
+
+const schema = yup.object().shape({
+  type: yup.string().required('Type is required'),
+  what: yup.string().required('What is required'),
+  why: yup
+    .array()
+    .min(1, 'At least one Why is required')
+    .required('Why is required')
+    .of(
+      yup.object().shape({
+        type: yup.string().required('Why Type is required'),
+        explain: yup
+          .string()
+          .required('Why Explain is required')
+          .when('type', {
+            is: ChangeRequestReason.OtherProject,
+            then: yup.string().test('wbs-num-valid', 'WBS Number is not valid', wbsTester)
+          })
+      })
+    )
+});
+
+const ProjectCreateChangeRequestForm: React.FC<ProjectCreateChangeRequestFormProps> = ({
+  onSubmit,
+  open,
+  onClose,
+  projectEdits
+}) => {
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+    register,
+    watch
+  } = useForm({
+    resolver: yupResolver(schema),
+    defaultValues: {
+      what: '',
+      why: [{ type: ChangeRequestReason.Other, explain: '' }],
+      type: ChangeRequestType.Issue
+    }
+  });
+
+  const permittedTypes = Object.values(ChangeRequestType).filter(
+    (t) => t !== ChangeRequestType.Activation && t !== ChangeRequestType.StageGate
+  );
+  const { fields: whys, append: appendWhy, remove: removeWhy } = useFieldArray({ control, name: 'why' });
+
+  const projectOptions: { label: string; id: string }[] = [];
+
+  const renderReasonInput = (index: number) => {
```

Comment:
this looks like a ton of repeat code from the actual create change request form, you should abstract absolutely as much as you possibly can between the two forms, honestly you should be using the actual create change request form but just dont force any proposed solutions